### ISSUE TITLE
extract id3 metadata from mp3s to display on tracks being played

### ISSUE THREAD

**Issue by psenough** [enoughrec/arecordlabel#8]

**Comment by dazld** [comment]
could maybe proxy the requests for tracks somehow.. but would be a lot of bandwidth.

**Comment by psenough** [comment]
technically speaking we could do a script to process them all, store the metadata somewhere in local files and then only use those. save the bandwidth. and keeping that in mind i guess it doesn't need to be an online process at all, could just run a script on local directories of the releases

**Comment by dazld** [comment]
could also do that, might be a lot more data to send with the rest of the bundle though.

**Comment by psenough** [comment]
did a little googling for javascript id3 extractors, there are a few options around:
https://github.com/43081j/id3
https://github.com/aadsm/JavaScript-ID3-Reader
just a matter of trying to integrate it with the player somehow

**Comment by dazld** [comment]
yeah, they all depend on the data being served with CORS headers, or you can't do binary stuffs on them.
